common-tools/clas-io/src/main/java/org/jlab/io/evio/EvioDataEventHandler.java

```diff
@@ -30,7 +30,7 @@
     public EvioDataEventHandler(byte[] buffer, ByteOrder b_order){
         evioBuffer = ByteBuffer.wrap(buffer);
         evioBuffer.order(b_order);
-        if(buffer.length>500*1024){
+        if(buffer.length>2500*1024){
```

Comment:
Is this new increased limit compatible with real data decoding? I thought the 500 kB was set to avoid issues with decoding of very large events. In any case, the printout should be updated accordingly.